html/user-activation/activation-trigger-keyboard-enter.html

```diff
@@ -15,15 +15,16 @@ <h1>Test for keyboard activation trigger for ENTER key</h1>
   <ol id="instructions">
     <li>Press ENTER key.
   </ol>
-  <script>
+  <script defer>
```

Comment:
Shouldn't the browsers ignore this `defer` because this usage is incompatible with the [spec](https://html.spec.whatwg.org/multipage/scripting.html#attr-script-defer)?  If this is not the case, I would suggest discussing this in an HTML issue before using in the tests.

Reply:
Oh you are right! They need to have a `src` 🤦. 

Ok, let me then go back to using onload=